Vic2ToHoI4/Source/OutHoi4/Operations/OutOperation.cpp

```diff
@@ -0,0 +1,125 @@
+#include "OutOperation.h"
+
+
+
+std::ostream& HoI4::operator<<(std::ostream& out, const Operation& operation)
+{
+	out << operation.getName() << " = { ### " << operation.getPriority() << " " << operation.getOperatives() << "/"
```

Comment:
Commented out first line is intended?

Reply:
Yep. It's more-or-less matching some presumably debug information in the original file.

More-or-less because the comments in the original file got out of date with the actual data, as comments do.